apps/src/templates/instructions/TeacherFeedback.jsx

```diff
@@ -161,6 +171,68 @@ export class TeacherFeedback extends Component {
     );
   }
 
+  renderFeedbackTeacherHeader() {
```

Comment:
(minor) would `renderCommentAreaTeacherHeader` or perhaps `renderCommentAreaHeaderForTeacher` be a bit clearer (if not very wordy)?  (I think I was confused because I wasn't sure what Feedback referred to.)